apps/language_server/lib/language_server/providers/completion.ex

```diff
@@ -246,9 +246,24 @@ defmodule ElixirLS.LanguageServer.Providers.Completion do
         "module"
       end
 
+    kind =
+      case subtype do
+        :behaviour -> :interface
+        :protocol -> :interface
+        :struct -> :struct
```

Comment:
exception as a struct?

Reply:
Yeah, it makes sense to return exception as a struct. I don't see anything that would map well to mix tasks, so I think it makes sense to leave it as a module.